slp_mtmt/slp_mtmt/parse/mtmt_trustzone_parser.py

```diff
@@ -27,11 +31,13 @@ def parse(self):
 
     def create_trustzone(self, border: MTMBorder) -> Trustzone:
         mtmt_type = self.__calculate_otm_type(border)
+        representations = calculate_representation(border, self.diagram_representation)
```

Comment:
Can we call the static method directly from the class, like in mtmt_component_parser line 33, to avoid introducing a variable used only in one place?

Reply:
I've done for legibility. The cost of having this var is not so expensive, but I agree we should have in the same way in both classes. I change it.